packages/glibc_lib237.rb

```diff
@@ -32,7 +33,7 @@ def self.install
     @filelist = File.readlines(@filelist_path, chomp: true).sort
 
     @filelist.each do |filename|
-      next unless filename.include?('.so') || filename.include?('bin/')
+      next unless (filename.include?('.so') || filename.include?('bin/')) && File.file?(filename)
```

Comment:
Should `File.file?(filename)` be the first comparison?
```
next unless File.file?(filename) && (filename.include?('.so') || filename.include?('bin/'))
```

Reply:
Good point.

Reply:
Is it better to do a string comparison before we look for a file on disk?

Reply:
(Because the string comparison is faster than checking to see if the file is there?)